docs-internal/Dependencies.md

```diff
@@ -0,0 +1,148 @@
+# Dependencies
+
+## Greenkeeper
+
+We have setup [greenkeeper][greenkeeper-home] to take care of updating dependencies to their latest
+versions.
+
+The configuration file is located at `greenkeeper.json`.
+
+## Lock files
+
+We are using [npm shrinkwrap][shrinkwrap-home] for the reasons described [here][shrinkwrap-issue].
+
+### Regenerate the lockfiles
```

Comment:
:man_shrugging: 
I shall open an issue over at npm to understand why creating these lockfiles is not deterministic...